src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.tsx

```diff
@@ -95,14 +128,6 @@ interface InternalMultiSelectProps<T extends MultiSelectData> {
   onRemoveAll?: () => void;
   /** Text for screen reader to indicate how many items are selected */
   ariaSelectedAmountText: string;
```

Comment:
Maybe we should add function also for the `ariaSelectedAmountText`.

Reply:
Added now

Reply:
Nice! 👍